lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -637,6 +645,14 @@ public void OnNext(IFailedTask failedTask)
                         return;
                     }
 
+                    if (_taskManager != null && _taskManager.IsMasterTaskCompleted())
+                    {
+                        Logger.Log(Level.Info,
+                           "Master task have been completed. So ignoring the failedTask {0} failure and execute DoneAction.", failedTask.Id);
```

Comment:
`Master task has completed successfully. Ignoring task {0} failure and completing the job.`